scripts/std-lib-regression.sh

```diff
@@ -70,9 +70,10 @@ export RUSTC_LOG=error
 RUST_FLAGS=(
     "--kani-compiler"
     "-Cpanic=abort"
+    "-Zalways-encode-mir"
```

Comment:
What does this do?

Reply:
This stores the MIR body of every function as part of the `.rlib` file. We use this option to build the sysroot as described [here](https://model-checking.github.io/kani/rfc/rfcs/0001-mir-linker.html#kanis-sysroot).